### ISSUE TITLE
PhilipJFrys client goes nuts v1.1.4

### ISSUE THREAD

**Issue by 1-alex98** [FAForever/downlords-faf-client#1640]
https://mega.nz/#!srZT1ShQ!M7qgTRm0gdNyPq28NnCYSBKjnVyePK6JNkXpWAv0Z48
https://faforever.zulipchat.com/user_uploads/14389/VYoGrt7sR162a3qBXrHPKCZB/threaddump-1585470042231.tdump

**Wanna have the bug fixed quickly?**
Visit Issue hunt...
[![Issue hunt](https://github.com/BoostIO/issuehunt-materials/raw/master/v1/issuehunt-button-v1.svg?sanitize=true)](https://issuehunt.io/r/FAForever/downlords-faf-client)

**Comment by 1-alex98** [comment]
@PhilipJFryFAF was it just cpu usage or also did it freeze? Which tab where u in when it frooze? What did yu do beforehand?

**Comment by PhilipJFryFAF** [comment]
i didn't do anything
i just had the client open and noticed that my PC didn't react properly

after looking at the task manager i noticed that my cpu load is at 100% and the client was using like 70-80%

i didn't interact with the client at all during this

**Comment by PhilipJFryFAF** [comment]
so not sure if the tab was frozen or not

**Comment by PhilipJFryFAF** [comment]
![image](https://user-images.githubusercontent.com/24656673/77854881-2d55b180-71ed-11ea-9cf8-614ef75c4ef8.png)
running into the same issue again - massive cpu load due to downlords-faf-client

the client itself is technically still working 
i was able to change from the chat to the play tab and back but it is very slow (5-10 seconds delay)

again i'm not playing and the client was just sitting in the back

**Comment by PhilipJFryFAF** [comment]
heapdump: https://mega.nz/#!s6AAEKoZ!d9ZlvyXZzU6l-87bypaE0Xi07BF6PxVKv5l0X7n4Oik
threaddump: https://mega.nz/#!xjIC3QYJ!jbVOigtDUS_iXKrsqiJim6xP2_Fyo9OxEHsPvhG-PM8

**Comment by dmalukov** [comment]
have the same issue sometimes

**Comment by 1-alex98** [comment]
https://filebin.net/7g3xqjf2k8zb2ytf

**Comment by 1-alex98** [comment]
@dmalukov try the newest prerelease

**Comment by JeroenDeDauw** [comment]
I've been using this client since way before it got put on the FAF site. At some point, probably over a year ago, I had to stop using it since it _always_ ends up maxing out my CPU. Since then I have tried many new versions of the client, all with the same problem. No issues with the other FAF client.

TL;DR: this is not a new issue

**Comment by dmalukov** [comment]
with the new version the issue is not reproduced
